botocore/__init__.py

```diff
@@ -60,8 +60,17 @@ def emit(self, record):
 
 BOTOCORE_ROOT = os.path.dirname(os.path.abspath(__file__))
 
+
 # Used to specify anonymous (unsigned) request signature
-UNSIGNED = object()
+class _UNSIGNED(object):
```

Comment:
More of a nit than anything, but would it be reasonable to do:

```python
class UNSIGNED(object):
    ...

UNSIGNED = UNSIGNED()
```

That would remove the ``_UNSIGNED`` class from the namespace as well.